<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sanescan::ScanSettingsWidget</class>
 <widget class="QWidget" name="sanescan::ScanSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="l_scanners">
     <property name="text">
      <string>Scanners</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cb_scanners"/>
   </item>
   <item>
    <widget class="Line" name="line_scanners">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="grid_layout">
     <item row="0" column="1">
      <widget class="QComboBox" name="cb_scan_source"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="l_scan_source">
       <property name="text">
        <string>Scan source</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="l_scan_mode">
       <property name="text">
        <string>Scan mode</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="l_bit_depth">
       <property name="text">
        <string>Bit depth</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="l_resolution">
       <property name="text">
        <string>Resolution</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cb_scan_mode"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cb_bit_depth"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cb_resolution"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="vs_settings">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="hs_refresh_devices">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="b_refresh_devices">
       <property name="text">
        <string>Refresh devices</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
